src/Xamarin.Android.Build.Tasks/UnitTests/Xamarin.ProjectTools/Xamarin.ProjectTools.csproj

```diff
@@ -0,0 +1,115 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project DefaultTargets="Build" ToolsVersion="4.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+	<Import Project="..\..\..\..\Configuration.props" />
```

Comment:
I wonder why this file is indented with tabs while the other `.csproj` files are indented with spaces, two spaces per indent, e.g. [Mono.Android.csproj](https://github.com/xamarin/xamarin-android/blob/c7a41e5/src/Mono.Android/Mono.Android.csproj).

Should it be fixed for consistency?